example/glide_computer.cpp

```diff
@@ -100,8 +100,9 @@ using namespace si::international::literals;
 using namespace si::unit_constants;
 
 template<QuantityKind QK1, QuantityKind QK2>
-constexpr Quantity auto operator/(const QK1& lhs, const QK2& rhs)
-  requires requires { lhs.common() / rhs.common(); } {
+constexpr Dimensionless auto operator/(const QK1& lhs, const QK2& rhs)
+  requires (!std::same_as<typename QK1::kind_type::base_kind, typename QK2::kind_type::base_kind>) &&
```

Comment:
`QuantityKindRelatedTo`?

Reply:
That's in `detail::`. Do I make it public?

Reply:
It seems it is useful so the user does not have to type such a long constraint.

Reply:
Done.